tests/terminaldetection/README.md

```
@@ -0,0 +1,113 @@
+### Run the Test
+
+From the repository root:
+
+```bash
+go test -v -run TestTerminalDetection
+```
+
+### Expected Output
+
+#### With Terminal
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)